release/test_git.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+source _git.sh
+source _liferay_common.sh
+source ../_test_common.sh
+
+function main {
+	set_up
+
+	if [ $? -eq "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}" ]
+	then
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	test_generate_release_notes
+
+	tear_down
+}
+
+function set_up {
+	export _PRODUCT_VERSION="2024.q2.0"
+	export _RELEASE_ROOT_DIR="${PWD}"
+	export LIFERAY_RELEASE_PRODUCT_NAME="dxp"
+
+	export _PROJECTS_DIR="${_RELEASE_ROOT_DIR}"/../..
+
+	export _BUILD_DIR="${_PROJECTS_DIR}/liferay-portal-ee"
+
+	if [ ! -d "${_PROJECTS_DIR}/liferay-portal-ee" ]
+	then
+		echo "The directory ${_PROJECTS_DIR}/liferay-portal-ee does not exist."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	lc_cd "${_PROJECTS_DIR}"/liferay-portal-ee
+
+	git checkout master &>/dev/null
+
+	git fetch upstream --no-tags &>/dev/null
+
+	git merge upstream/master &>/dev/null
```

Comment:
@kiwm, will it for sure merge the downloaded `upstream/master` branch into the local `master` branch?